src/middleware/auth.middleware.js

```diff
@@ -0,0 +1,35 @@
+const jwt = require('jsonwebtoken');
+require('dotenv').config();
+
+module.exports = (req, res, next) => {
+    console.log('hear');
+    const authHeader = req.get('Authorization');
+    if (!authHeader) {
+        const error = new Error('Authorization header missing!');
+        error.statusCode = 401;
```

Comment:
don't add error messages like this throw new Error("Authorization header missing!");